### PR TITLE
chore: Add toast hight order component

### PR DESCRIPTION
**Pull request by Wandarry**
### Child PR

- [x] PR #34 

Add toast hight order component

### CODE REVIEW

File: helpers/emitters.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { TinyEmitter } from "tiny-emitter";
+
+// the name of toast event
```
Comment (BOCOVO) [inline helpers/emitters.ts]: Remove this comment. The line below too.

File: app/_layout.tsx [hunk 2]
```diff
@@ -48,6 +49,7 @@ function Wrapper({ children }: PropsWithChildren) {
   return (
     <ThemeProvider value={colorScheme === "dark" ? DarkTheme : DefaultTheme}>
       <GluestackUIProvider config={extendedThemeConfig}>
+        <ToastEventHandler />
```
Comment (BOCOVO) [inline app/_layout.tsx]: I'm wondering if we shouldn't call it `ToastCreator`. What do think about?

**Review by BOCOVO — CHANGES_REQUESTED** [review]

File: app/_layout.tsx (hunk 2 above)
Comment (Wandarry) [reply app/_layout.tsx]: Why not ? It's more understandable

**Comment by Wandarry** [comment]
@BOCOVO c'est bon ici

**Review by BOCOVO — APPROVED** [review]

**Pull request merged**